static/app/components/forms/model.tsx

```diff
@@ -34,6 +34,10 @@ export type FormOptions = {
    * Initial form data
    */
   initialData?: Record<string, FieldValue>;
+  /**
+   * Custom transformer function to map error response JSON to form errors
+   */
+  mapFormErrors?: (responseJSON?: any) => any;
```

Comment:
should we maybe use the parameter type and `ReturnType` of the function `mapFormErrors`, so in the future if types are added there it will be reflected here?